jwala-services/src/main/java/com/cerner/jwala/service/group/GroupStateNotificationService.java

```diff
@@ -1,6 +1,11 @@
 package com.cerner.jwala.service.group;
```

Comment:
PLEASE IGNORE THIS COMMIT. For some reason this commit (which is in another branch) got merged in this branch.